lib/generators/react_on_rails/templates/base/server_rendering/client/webpack.server.rails.config.js

```diff
@@ -2,15 +2,29 @@
 
 const webpack = require('webpack');
 const path = require('path');
+const React = require('react');
 
 const devBuild = process.env.NODE_ENV !== 'production';
 const nodeEnv = devBuild ? 'development' : 'production';
 
+const serverEntry = ['./app/bundles/HelloWorld/startup/serverGlobals', 'react'];
+const serverLoaders = [
+      {test: /\.jsx?$/, loader: 'babel-loader', exclude: /node_modules/},
+
+      // React is necessary for the client rendering:
+      {test: require.resolve('react'), loader: 'expose?React'},
+    ];
+
+if (React.version >= '0.14') {
```

Comment:
@yorzi @robwise @alexfedoseev @samnang @rstudner @jbhatab How about if we just include the react 0.14 syntax for the generator?

If somebody intentionally does not want react 0.14, then they will know to remove react-dom from this file as it will fail fast.

I prefer not to put this logic in the config file.